src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  box-sizing: border-box;
```

Comment:
Would be better to move it to main and apply the property to * or move it `body`